website/static/js/logTextParser.js

```diff
@@ -25,6 +25,12 @@ function ravenMessage (message, logObject) {
     }
 }
 
+function toRelativeUrl(url) {
+    var parser = document.createElement('a');
```

Comment:
@brianjgeiger Also, would any of the log links be an external link? Currently I am converting all the links to relative assuming that all the log links are internal links.